pkg/hal/cli/capability/create.go

```diff
@@ -0,0 +1,246 @@
+package capability
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"gopkg.in/AlecAivazis/survey.v1"
+	"halkyon.io/api/capability/v1beta1"
+	halkyon "halkyon.io/api/v1beta1"
+	"halkyon.io/hal/pkg/cmdutil"
+	"halkyon.io/hal/pkg/k8s"
+	"halkyon.io/hal/pkg/log"
+	"halkyon.io/hal/pkg/ui"
+	"halkyon.io/hal/pkg/validation"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ktemplates "k8s.io/kubectl/pkg/util/templates"
+	"strings"
+	"time"
+)
+
+const createCommandeName = "create"
+
+type createOptions struct {
+	category    string
+	subCategory string
+	version     string
+	paramPairs  []string
+	parameters  []halkyon.NameValuePair
+	name        string
+}
+
+var (
+	capabilityExample = ktemplates.Examples(`  # Create a new database capability de type postgres 10 and sets up some parameters as the name of the database and the user/password to connect.
```

Comment:
_Create a new database capability **of** type postgres 10 and sets up some parameters as the name of the database and the user/password to connect._